.github/workflows/quality-assurance.yml

```diff
@@ -67,14 +65,19 @@ jobs:
       - name: Check Commit Signature
         run: python .github/scripts/check_commit_signature.py
 
+      - name: Debug github ref
+        run: |
+          echo ${{ github.ref }}
+          echo ${{ github.ref_name }}
+
       - name: Check Newsfragment
         if: |
-          ${{
-            startsWith(github.ref, 'refs/pull/')
-            && !(startsWith(github.ref_name, 'dependabot')
-              || startsWith(github.ref_name, 'yolo')
-              || startsWith(github.ref_name, 'release'))
-          }}
+          startsWith(github.ref, 'refs/pull/')
+          && !(
```

Comment:
Given we already do the release branch check within the script, shouldn't we also migrate all those checks in the scripts ? (maybe only leaving the `startsWith(github.ref, 'refs/pull/')`)

Reply:
we could add them to the script but I would not remove them from the conditional expression, It faster that executing the script

Reply:
speed difference is minimal, on the other hand it's better for readability have every branch check at the same place (see my own PR to fix this issue: I only saw the check were done in the script and missed what is in this file)